### ISSUE TITLE
Colorblind-friendly Preview colors

### ISSUE THREAD

**Issue by JRJae** [bambulab/BambuStudio#4824]
As a colorblind person, sometimes it's hard to distinguish the colors in the preview. "Lyne type" preview is good enough, but I have a lot of issues when visualizing "Speed", "Flow", or any other smooth color transition. In this example, I'm unable to distinguish any speed from 90 to 250 mm/s:

<img width="758" alt="Captura de pantalla 2024-09-15 a las 10 04 37" src="https://github.com/user-attachments/assets/04568406-762d-4b73-bf76-fd1eb77684e4">

There are many studies about which color palettes or ranges can be used for this kind of graphical representation ([here's just an example](https://www.datylon.com/blog/data-visualization-for-colorblind-readers)).

It would be great if a different color palette could be chosen for the preview mode.

Thanks in advance for your time and the great work you are doing 😉

**Comment by thieri** [comment]
Yes, would be great, the contrast between the colors is SOOOOOO small, i cant discern them. Maybe different gradients, or clickable to hide some speeds/flows/regions... like for line type.
